Dockerfile

```diff
@@ -1,7 +1,7 @@
 # syntax=docker/dockerfile-upstream:master
 
-ARG RUNC_VERSION=v1.1.14
-ARG CONTAINERD_VERSION=v1.7.22
+ARG RUNC_VERSION=v1.1.15
+ARG CONTAINERD_VERSION=v1.7.23
```

Comment:
Should we keep 1.7.22 if we can't vendor the go module?

Reply:
They don't need to be the same. We are still compatible with v1.7.23 daemon.